notebooks/quick-look.ipynb

```diff
@@ -232,10 +232,10 @@
     }
    ],
    "source": [
-    "from rubicon_ml.ui import Dashboard\n",
+    "from rubicon_ml.viz import Dashboard\n",
     "\n",
     "\n",
-    "Dashboard(rubicon=rubicon).run_server_inline()"
+    "Dashboard(project.experiments()).show()"
    ]
   },
```

Comment:
the quicklook still has a screenshot of the old dashboard, could we update the screenshot with one of the new dashboard or just remove it all together?

Reply:
good catch! I'll update the images